dags/bqetl_desktop_mobile_search_aggregates_monthly.py

```diff
@@ -0,0 +1,87 @@
+# Generated via https://github.com/mozilla/bigquery-etl/blob/main/bigquery_etl/query_scheduling/generate_airflow_dags.py
+
+from airflow import DAG
+from airflow.sensors.external_task import ExternalTaskMarker
+from airflow.sensors.external_task import ExternalTaskSensor
+from airflow.utils.task_group import TaskGroup
+import datetime
+from utils.constants import ALLOWED_STATES, FAILED_STATES
+from utils.gcp import bigquery_etl_query, gke_command
+
+docs = """
+### bqetl_desktop_mobile_search_aggregates_monthly
+
+Built from bigquery-etl repo, [`dags/bqetl_desktop_mobile_search_aggregates_monthly.py`](https://github.com/mozilla/bigquery-etl/blob/main/dags/bqetl_desktop_mobile_search_aggregates_monthly.py)
+
+#### Description
+
+Populate monthly aggregates based on daily clients table for desktop
+#### Owner
+
+akommasani@mozilla.com
+"""
+
+
+default_args = {
+    "owner": "akommasani@mozilla.com",
+    "start_date": datetime.datetime(2019, 1, 1, 0, 0),
+    "end_date": None,
+    "email": ["telemetry-alerts@mozilla.com", "akommasani@mozilla.com"],
+    "depends_on_past": False,
+    "retry_delay": datetime.timedelta(seconds=1800),
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "retries": 2,
+}
+
+tags = ["impact/tier_1", "repo/bigquery-etl"]
+
+with DAG(
+    "bqetl_desktop_mobile_search_aggregates_monthly",
+    default_args=default_args,
+    schedule_interval="0 5 2 * *",
+    doc_md=docs,
+    tags=tags,
+) as dag:
+
+    search_derived__desktop_mobile_monthly_search_aggregates__v1 = bigquery_etl_query(
+        task_id="search_derived__desktop_mobile_monthly_search_aggregates__v1",
+        destination_table="desktop_mobile_monthly_search_aggregates_v1",
+        dataset_id="search_derived",
+        project_id="moz-fx-data-shared-prod",
+        owner="akommasani@mozilla.com",
+        email=["akommasani@mozilla.com", "telemetry-alerts@mozilla.com"],
+        date_partition_parameter="submission_date",
+        depends_on_past=False,
+    )
+
+    wait_for_search_derived__mobile_search_clients_daily__v1 = ExternalTaskSensor(
+        task_id="wait_for_search_derived__mobile_search_clients_daily__v1",
+        external_dag_id="bqetl_mobile_search",
+        external_task_id="search_derived__mobile_search_clients_daily__v1",
+        execution_delta=datetime.timedelta(seconds=79200),
```

Comment:
Do we need these task sensors?

Reply:
These were added by default while generating the dag, believe these are required for the search_daily table to be populated (dependency)

Reply:
Hmm, they're not really doing much since we would actually want the entire past month to be complete. It might be possible to remove them by adding an empty `referenced_tables`